.travis.yml

```diff
@@ -9,9 +9,7 @@ node_js:
 branches:
   only:
     - master
-    - /^v(0|[1-9]\d*)\.(0|[1-9]\d*)$/
```

Comment:
Why have we removed those?

Reply:
We don't have such branches. Only `v*.*-dev`.